challenges/objects-arrays.js

```diff
@@ -45,23 +64,42 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 /* Request 1: Create a new array called universities that contains all the univeristies in the graduates array.  
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
-const universities = [];
-console.log(universities)
+const universities = graduates.map (function(item){
```

Comment:
Make sure you read _all_ the instructions/comments, you were NOT supposed to use array methods here.